### PR TITLE
fixing and adding GRAMENE_PATHWAY links for plants

### PR DESCRIPTION
**Pull request by dbolser-ebi**
based on the list available here http://pathway.gramene.org/

### CODE REVIEW

**Comment by dbolser-ebi** [comment]
The funny thing is, I don't see this data showing up anywhere in the interface, however, I figure what config we do have should at least be consistent with what's available...

Any clue where this should appear and how to configure it to appear there? i.e. I'm assuming ###ID### is a gene_id, but it could be a translation_id...

File: conf/ini-files/Oryza_brachyantha.ini [hunk 1]
```diff
@@ -26,7 +26,6 @@ DATABASE_USERDATA = Oryza_brachyantha_userdata
 ; DO NOT REMOVE THIS HEADER - defined in DEFAULTS.ini
 
 [ENSEMBL_EXTERNAL_URLS]
-GRAMENE_PATHWAY = http://pathway.gramene.org/BRACHY/NEW-IMAGE?type=REACTION&object=###ID###
```
Comment (dbolser-ebi) [inline conf/ini-files/Oryza_brachyantha.ini]: This was just wrong.

**Comment by nicklangridge** [comment]
The link templates are used for linking to corresponding external dbs, e.g. http://plants.ensembl.org/Arabidopsis_thaliana/Gene/Matches?g=AT3G52430;r=3:19431371-19434403;t=AT3G52430.1

They will only appear if you have appropriate xrefs.

**Comment by dbolser-ebi** [comment]
Ta.

**Pull request merged**